docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.4'
+
+services:
+  site:
+    # Recommended docker image by Hugo docs:
+    #    https://gohugo.io/getting-started/installing/#docker
+    #
+    # TODO: Use a minimal image instead. The alpine version
+    # "klakegg/hugo:0.101.0-ext-alpine" complains:
+    #
+    #    fatal: unsafe repository ('/src' is owned by someone else)
+    #
+    # This is because Git 2.35.2 added this check as a security fix. The
+    # non-minimal image has an older version of Git, so it doesn't complain.
+    # Can be fixed with a .gitconfig containing:
+    #
+    #     [safe]
+    #         directory = /home/repon
+    image: "klakegg/hugo:0.101.0-ext"
```

Comment:
question: I do not use docker compose so am wondering, how stable is this image uri?  Do we expect this to continue to work, say, next year?  If not is there another URI form that would map to "latest" or something?

Reply:
Good call, Docker image tagging conventions do have a concept of `latest`. Tags are free-form like git tags. In this repo, `latest` itself corresponds to a non-"extended" version of Hugo, but we want "extended". The README for this image repository (https://hub.docker.com/r/klakegg/hugo/) describes a `latest-ext` tag that looks exactly like what we want.

Reply:
:heavy_check_mark: 96711dd

Reply:
Forgot to answer one of your questions:

> Do we expect this to continue to work, say, next year?

I would expect `0.101.0-ext` to continue to work as long as someone is pulling that version of the image from DockerHub every month or so. I forget the exact policy, but if you don't pay DockerHub, they will archive your images that haven't been pulled in a certain amount of time. A `latest` image is definitely safer against this possibility.